src/tallies/TallyBase.C

```diff
@@ -80,6 +81,9 @@ TallyBase::validParams()
 
   params.registerBase("Tally");
   params.registerSystemAttributeName("Tally");
+  params.addParam<bool>("use_displaced_mesh",
```

Comment:
you automatically inherit this in the daughter classes. Apply same feedback here